docs/admin/infrastructure/google/credentials.rst

```diff
@@ -0,0 +1,36 @@
+##################################
+Getting GKE Kubernetes credentials
+##################################
+
+In order to use the standard Kubernetes administrative command :command:`kubectl` or other commands built on the same protocol (such as Helm_ or the Phalanx installer), you must first have stored authentication credentials for the target Kubernetes cluster.
+Google provides a mechanism to obtain those credentials using the :command:`gcloud` command.
+Here are the steps:
+
+#. Ensure you have a Google account with access to the Google Cloud Platform project where your target Kubernetes cluster is running.
+   For Phalanx environments run by SQuaRE, this access must be via an ``lsst.cloud`` Google account that is used only for Rubin activities.
+   If you do not already have such an account or permissions and need administrative access to a Phalanx environment maintained by SQuaRE, contact SQuaRE for access.
+
+#. `Install gcloud <https://cloud.google.com/sdk/docs/install>`__ on the system on which you want to run privileged Kubernetes commands.
+
+#. `Initialize gcloud <https://cloud.google.com/sdk/docs/initializing>`__.
+   You will need to have access to the Google Cloud Platform project where your target Kubernetes cluster is running.
+
+   If you have access to multiple Google Cloud Platform projects, you will be asked to select one as your default project.
+   You may wish to choose the project for the Phalanx environment you use most often.
+   You can find the project ID of a Phalanx project hosted on GKE in its :doc:`environments page </environments>`.
+
+#. `Install kubectl and the GKE auth plugin <https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl>`__.
+   As part of that installation, you will run the :command:`gcloud` command that obtains credentials usable by :command:`kubectl` and other privileged Kubernetes commands.
+   See below for assistance with the precise :command:`gcloud` command to run.
```

Comment:
You might be able to drop this sentence because I'm not sure where 'below' we're referring to. I'm guessing the environments pages? But might be fine to leave out.